views/homework/components/overview_card_template.hbs

```diff
@@ -74,7 +74,7 @@
         {{#if ../this.isTeacher}}
         {{#each ../this.actions}}
         {{#if this.method}}
-        <a class="btn btn-secondary btn-sm {{this.class}}" href="{{this.link}}" target="_blank"
+        <a class="btn btn-secondary btn-sm {{this.class}}" href="{{this.link}}" redirect="?p={{@root.pagination.currentPage}}" target="_blank"
```

Comment:
where is this `redirect` used?

Reply:
it is used in the delete modal to redirect to the current page after deletion